tuned/plugins/plugin_selinux.py

```diff
@@ -9,7 +9,27 @@
 
 class SelinuxPlugin(base.Plugin):
 	"""
-	Plugin for tuning SELinux options.
+	`selinux`::
+	
+	Plug-in for tuning SELinux options.
+	+
+	SELinux decisions, such as allowing or disallowing access, are
+	cached. This cache is known as the Access Vector Cache (AVC). When
+	using these cached decisions, SELinux policy rules need to be checked
+	less, which increases performance. The [option]`avc_cache_threshold`
+	option allows adjusting the maximum number of AVC entries.
+	+
+	NOTE: Prior to changing the default value, evaluate the system
+	performance with care.	Blindly increasing the value could potentially
```

Comment:
Please remove the word 'Blindly' and change the beginning of the sentence to 'Increasing the'